README.md

```diff
@@ -0,0 +1,16 @@
+
+```
+
+oc login -u system:admin
+make cluster/clean
+make cluster/prepare
+
+make code/run
+
+
+TODO:
+oc create clusterrole mobileclient-admin --verb=create,delete,get,list,patch,update,watch --resource=mobileclients
+oc adm policy add-cluster-role-to-user mobileclient-admin system:serviceaccount:mdc:example-mobiledeveloperconsole 
```

Comment:
This should apparently be `oc adm policy add-cluster-role-to-user mobileclient-admin system:serviceaccount:mdc:mobile-developer-console-operator` (wrong SA name).

Reply:
we need to do this within the reconciliation anyway.
thanks for pointing this